recipes/django-nested-inline/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.3.5" %}
+
+package:
+    name: django-nested-inline
+    version: {{ version }}
+
+source:
+    fn: django-nested-inline-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/d/django-nested-inline/django-nested-inline-{{ version }}.tar.gz
+    md5: e59b87c90a0eebcccf902ae243121578
+    patches:
+        - zipsafe.patch
```

Comment:
@kwilcox if I recall the reason we have that patch is not to "fix" django-nested-inline, but because you needed the uncompressed files. Correct? So that won't make its way upstream.